server/db/mongoose.js

```diff
@@ -0,0 +1,7 @@
+const mongoose = require("mongoose");
+
+mongoose.connect(process.env.MONGO_URL, {
```

Comment:
it looks like I have to add `require("dotenv").config();` to this file get this to access the dotenv file properly (not sure if it's just me?)

Reply:
Oh I'm not sure about that. Did you run npm install before testing the changes I made? The dev2 scripts makes use of the env-cmd package which I installed yesterday.

Reply:
hmm I did do npm install in the main directory + server again, we can just wait for someone else to approve. it works for someone else, then maybe something is weird going on with mine

Reply:
I also require
`require("dotenv").config();`

I did both an npm install and npm install --force for both the server and main

Reply:
Yep, also need to add `require("dotenv").config();`
I think the dev2 script is just a workaround, but we should probably use include the `require`

Reply:
Ok I'll just remove that second dev script for now. I've added `require("dotenv").config();`